dockerfiles/alpine/broker/Dockerfile

```diff
@@ -16,45 +16,53 @@
 #
 # ------------------------------------------------------------------------
 
-# set base Docker image to AdoptOpenJDK Alpine Docker image
-FROM adoptopenjdk/openjdk8:jdk8u192-b12-alpine
-MAINTAINER WSO2 Docker Maintainers "dev@wso2.org"
+FROM adoptopenjdk/openjdk8:jdk8u212-b03-alpine
```

Comment:
@Vathsan it seems like this AdoptOpenJDK update level is outdated. Shall we please update to [this](https://hub.docker.com/layers/adoptopenjdk/openjdk8/jdk8u282-b08-alpine/images/sha256-8ff8aa92638ada9e78a6f06aa1308f6ce98ef3092c2157dbe8a7bd6807689f04?context=explore) latest version? Please apply the same image tag to the other Alpine images as well.

Reply:
Updated AdoptOpenJDK version to **adoptopenjdk/openjdk8:jdk8u282-b08-alpine** in all Alpine images